test/e2e/util/util.go

```diff
@@ -624,7 +678,7 @@ func EnsureAllRoutesUseHCPRouter(t *testing.T, ctx context.Context, hostClient c
 		}
 		for _, route := range routes.Items {
 			original := route.DeepCopy()
-			ingress.AddHCPRouteLabel(&route)
+			ingress.AddRouteLabel(&route)
```

Comment:
why is this being renamed?

Reply:
My bad, tbh dunno in which moment I've changed that part xD

Reply:
Ohh I think it was because I've needed a rebase. Solved